benchmarks/modules/cfd/pages/pipestokes/README.adoc

```diff
@@ -26,15 +26,18 @@ with stem:[\boldsymbol{\mu}] is the dynamic viscosity, stem:[\boldsymbol{p}] is
 
 === Initial conditions
 * the fact that we are not considering the time evolution in this case, 
-we have stem:[v_{inlet} = D y(height-y)] 
-To determine D, we know that for stem:[y=\frac{height}{2}] we have the maximal velocity, so 
-stem:[D=\frac{v_{max}}{\frac{height}{2}\left(height-\frac{height}{2}\right)}]
+we have stem:[v_{inlet} = D y(height-y)]. To determine D, we know that for stem:[y=\frac{height}{2}] we have the maximal velocity, so 
 
-* in this case, there is no source term so stem:[f=0].
+[stem]
+++++
+D=\frac{v_{max}}{\frac{height}{2}\left(height-\frac{height}{2}\right)}
+++++
+
+* in this case, there is no source term so, stem:[f=0].
 
 === Boundary conditions
-* On wall,a homogenous Dirichlet condition stem:[\boldsymbol{u}=0]
-* On outlet, a Newmann condition stem:[\boldsymbol{\sigma}\boldsymbol{n}=0], where stem:[\sigma=-pI_d+2\mu D(\boldsymbol{u})] and stem:[D(\boldsymbol{u})=\frac{1}{2}(\nabla \boldsymbol{u}+\nabla \boldsymbol{u}^{T})],  stem:[\sigma ]  is the contraints tensor and stem:[D] is the deformation tensor.
+* On wall, a homogenous Dirichlet condition stem:[\boldsymbol{u}=0]
```

Comment:
markers should have specific fonts modifiers, eg typewrite `wall` or bold `**wall**`  so that they are quickly identified. the rationale is that they are words in the sentence but identifiers/nouns